WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Print Shipping Label/PrintShippingLabelViewController.swift

```diff
@@ -185,21 +201,69 @@ private extension PrintShippingLabelViewController {
             configurePaperSizeOptions(cell: cell)
         case let cell as ImageAndTitleAndTextTableViewCell where row == .printingInstructions:
             configurePrintingInstructions(cell: cell)
+        case let cell as ButtonTableViewCell where row == .printButton:
+            configurePrintButtonRow(cell: cell)
+        case let cell as ButtonTableViewCell where row == .saveButton:
+            configureSaveButton(cell: cell)
         default:
             break
         }
     }
 
+    func rowsToDisplay() -> [Row] {
+        var rows: [Row?]
+        switch printType {
+        case .print:
+            rows = [
+                .spacerBetweenHeaderCells,
+                .headerText,
+                .spacerBetweenHeaderCells,
+                .headerImage,
+                .spacerBetweenHeaderCells,
+                .paperSize,
+                .spacerBetweenPaperSizeSelectorAndInfoLinks,
+                .printButton,
+                .saveButton,
+                .paperSizeOptions,
+                .printingInstructions
+            ]
+        case .reprint:
+            rows = [
+                .headerText,
+                .infoText,
+                .spacerBetweenInfoTextAndPaperSizeSelector,
+                .paperSize,
+                .spacerBetweenPaperSizeSelectorAndInfoLinks,
+                .paperSizeOptions,
+                .printingInstructions
+            ]
+        }
+        return rows.compactMap { $0 }
```

Comment:
nit: it seems that we can't have `nil` values, so the `compactMap` should be superfluous here.